types/dockerode/index.d.ts

```diff
@@ -1129,6 +1129,7 @@ declare namespace Dockerode {
 
     interface ContainerCreateOptions {
         name?: string | undefined;
+        platform?: string | undefined;
```

Comment:
Per the latest docker API docs (https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerCreate), this is an allowed query parameter, which dockerode supports (see https://github.com/apocas/dockerode/issues/721#issuecomment-1520328035)